csharp/src/Apache.Arrow/Ipc/ArrowTypeFlatbufferBuilder.cs

```diff
@@ -98,7 +98,10 @@ public void Visit(BinaryType type)
 
             public void Visit(ListType type)
             {
-                throw new NotImplementedException();
+                Flatbuf.List.StartList(Builder);
```

Comment:
Do we need to do anything with the `ListType type` parameter here? Like Visit the "ValueField" child?

(sorry for my ignorance w.r.t. the ListType)

Reply:
I suppose we don't need to do anything because of reasons below.

* The `ArrowStreamWriter.GetChildrenFieldOffsets` method visits all children.
* [`List` in `Schema.fbs`](https://github.com/apache/arrow/blob/master/format/Schema.fbs#L47) has no members.
  * When deserializing `ListType` from `FlatBuffers`, the `ValueField` data is created from children of `Field`.
  * I referenced the [C++ imprementation](https://github.com/apache/arrow/blob/master/cpp/src/arrow/ipc/metadata_internal.cc#L317)

Reply:
Sounds good to me. Thanks for checking.